portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/Minified.testcase

```diff
@@ -0,0 +1,54 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Check for errors in the minified browser console"
+	@priority = "3"
+	test SmokeMinifiedEnabled {
+		property custom.properties = "minified.enabled=true";
```

Comment:
`property test.assert.warning.exceptions = "true";`

Add this property too so it will fail the test if liferay log warnings/errors are present.